doc/readme.html

```diff
@@ -423,6 +423,22 @@ <h4>Optional interfaces to other software packages</h4>
                 about compatibility and configuration can be found <a href=external-libs/cuda.html target="body">here</a>.
             </p>
 
+            <dt><a name="Ginkgo"/>
+                <a href="https://github.com/ginkgo-project/ginkgo" target="_top">Ginkgo</a></dt>
+            <dd>
+                <p>
+                    <a href="https://github.com/ginkgo-project/ginkgo" target="_top">Ginkgo</a>
+                    is a numerical linear algebra library with highly optimized kernels
+                    for many core architectures with focus on fine level parallelism.
+                    It allows for easy switching of the executing paradigm (cuda, omp, .etc)
+                    while providing a multitude of linear solvers and preconditioners
+                    and extension to other linear operators with no changes required in the code,
```

Comment:
> with no changes required in the code

This, as far as I understand, is not quite true. You've got the different `gko::Executor` that need to be configured. I think that the word "no" should be substituted by "minimal".